lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery.hs

```diff
@@ -42,9 +42,10 @@ module Cardano.Wallet.Primitive.AddressDiscovery
     , accountPubKey
     , mkAddressPool
     , lookupAddress
+    , indexedAddresses -- fixme: internals exposed for sql
```

Comment:
NOTE: `addresses` is already exposed and does exactly what the SQL engine needs I believe.